lib/perlfaq5.pod

```diff
@@ -1539,7 +1539,7 @@ a similar interface, but does the traversal for you too:
 (contributed by brian d foy)
 
 If you have an empty directory, you can use Perl's built-in C<rmdir>.
-If the directory is not empty (so, with files or subdirectories), you
+If the directory is not empty (so, no files or subdirectories), you
```

Comment:
This was just corrected (by you) in commit 9f411960. Why is it being reverted again?

Reply:
Sorry, Karen. I mess all this pull.. Sorry. I will try to repair it.

Reply:
Karen, Can you remove all this commits/branch? Next, I can to resend all the changes into a new pull request.

Reply:
No, you can fix it all yourself in this branch.  https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History  This is a very important skill to learn.

Reply:
Maybe the problem that I edited & pull the changes with github only. I don't have a cloned git repository of perlfaq at my computer. I use the option of github of "edit this file" (in other words, fork the perlfaq repository, edit file(s), create pull request, all within github web pages).